kv/local_sender.go

```diff
@@ -142,13 +142,28 @@ func (ls *LocalSender) Send(call client.Call) {
 // by consulting each store in turn via Store.LookupRange(key).
 // Returns RaftID and replica on success; RangeKeyMismatch error
 // if not found.
-func (ls *LocalSender) lookupReplica(start, end proto.Key) (int64, *proto.Replica, error) {
+// TODO(tschottdorf) with a very large number of stores, the LocalSender
+// may want to avoid scanning the whole map of stores on each invocation.
```

Comment:
Seems like `LocalSender` (and perhaps `Store`) should be using something like `kv.RangeCache` where an interval tree from key range to range is kept.